baseplate/frameworks/pyramid/__init__.py

```diff
@@ -309,39 +296,6 @@ def start_server_span(
         config.add_request_method(start_server_span, "start_server_span")
 
 
-def paste_make_app(_: Dict[str, str], **local_config: str) -> Any:
-    """Make an application object, PasteDeploy style.
-
-    This is a compatibility shim to adapt the baseplate app entrypoint to
-    PasteDeploy-style so tools like Pyramid's pshell work.
-
-    To use it, add a single line to your app's section in its INI file:
-
-        [app:your_app]
-        use = egg:baseplate
-
-    """
-    return make_app(local_config)
-
-
-def pshell_setup(env: Dict[str, Any]) -> None:
```

Comment:
Do you want to remove the reference to `pshell_setup` in the `includeme.start_server_span` comment?

Reply:
nice! tore that stuff out altogether and simplified startup as a result.